README.md

```diff
@@ -123,7 +131,17 @@ However, if you'd like to daemonize it, you have to add the flag `-d`:
 docker-compose up -d
 ```
 
-Then:
+#### Optionally purge existing installation
+
+If you already had the package installed, and want a clean start:
+
+```bash
+docker-compose exec web rm composer.lock
+docker-compose exec web rm -rf vendor/
+docker-compose exec web rm -rf build/
+```
+
+#### Installing the package
```

Comment:
Btw there is some inconsistency here: "Optionally purge ..." is in imperative form, "Running the tests" is in present participle form. Not sure what is preferable.

Reply:
Some interesting references
https://developers.google.com/style/headings
https://writing.stackexchange.com/questions/1545/verb-tense-for-technical-document-titles
https://ffeathers.wordpress.com/2013/10/12/death-of-the-gerund-in-technical-documentation/
https://idratherbewriting.com/2008/02/11/are-gerunds-in-topic-titles-problematic-in-search-results/

There is no final conclusion, but I find more sources that recommend the imperative or "bare infinitive".

Reply:
However, not every headline is the same:
- Some headlines describe steps in a sequence. Here the imperative form seems most suitable.
- Other headlines describe actions we can do again at any time, such as running the tests.

Still, having different verb forms in the same sequence of titles feels wrong, so I will go for consistency.